projects/js-packages/components/components/alert/index.tsx

```diff
@@ -19,9 +18,20 @@ export const LEVEL_WARNING = 'warning';
 export const LEVEL_INFO = 'info';
 export const LEVEL_SUCCESS = 'success';
 
-export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ];
+export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ] as const;
 
-const getIconByLevel = level => {
+type AlertProps = {
+	/** The severity of the alert. */
+	level: typeof LEVEL_ERROR | typeof LEVEL_WARNING | typeof LEVEL_INFO | typeof LEVEL_SUCCESS;
+
+	/** Children to be rendered inside the alert. */
+	children: React.ReactNode;
```

Comment:
You could remove this one since you're using `React.PropsWithChildren`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L799

Reply:
removed but restored again because I expect `children` to be required:

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/77539/166676596-2de77d2f-e5c9-4e34-9f66-6edde9037ff8.png">

[PropsWithChildren](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L799) sets children as optional.